docs/network/architecture.md

```diff
@@ -13,16 +15,22 @@ These components are instances of binaries in the [Octez software suite](https:/
 ## Sequencer
 
 Etherlink relies on a sequencer to publish transactions.
-It receives transactions from EVM nodes, puts them in order, and packages them into a blueprint, which is a group of Etherlink transactions.
+It receives transactions from EVM nodes, puts them in order, and packages them into a block.
+These blocks are specific to Etherlink and are separate from Tezos layer blocks.
+These blocks are also called "blueprints."
 
-Each blueprint contains:
+The sequencer generates blocks on demand.
+It generates a block at least every 6 seconds, even if the block is empty.
```

Comment:
```suggestion
It generates a block at a regular interval, even if the block is empty
```
Nit suggestion, this is not inherent to Etherlink, it's just the current configuration picked by the sequencer. That's my opinion thought, I understand that we would like to document this.

Similarly `500ms` is purely arbitrary and we could pick a very different values if we think it's more efficient.

Reply:
Updated to: 

> The sequencer generates blocks at a variable rate, depending on demand. Currently, it generates a block at least every 6 seconds, even if the block is empty. As demand increases, it generates blocks more quickly, up to a block every 500ms. These values can change with Etherlink upgrades.